frontend/src/modules/SimulationTimeSeries/view/utils/PlotBuilder.ts

```diff
@@ -274,7 +272,11 @@ export class PlotBuilder {
             }
         }
 
-        return this._figure.makePlot({ onClick: handleOnClick });
+        return {
+            data: this._figure.makeData(),
+            layout: this._figure.makeLayout(),
+            ...customProps,
+        };
```

Comment:
Why changing the pattern? Either continue to use `makeLayout` and `makeData` as in the other builders or switch overall to the `makeProps` strategy. `customProps` seems unnecessary here?